src/trajectory.jl

```diff
@@ -149,22 +151,25 @@ function transition(
     h::Hamiltonian,
     z::PhasePoint
 ) where {T<:Real}
+    H0 = energy(z)
     z′ = step(rng, τ.integrator, h, z, τ.n_steps)
     # Accept via MH criteria
-    is_accept, α = mh_accept_ratio(rng, energy(z), energy(z′))
+    is_accept, α = mh_accept_ratio(rng, H0, energy(z′))
     if is_accept
         # Reverse momentum variable to preserve reversibility
         z = PhasePoint(z′.θ, -z′.r, z′.ℓπ, z′.ℓκ)
     end
-    stat = (
-        step_size=τ.integrator.ϵ,
-        n_steps=τ.n_steps,
-        is_accept=is_accept,
-        acceptance_rate=α,
-        log_density=z.ℓπ.value,
-        hamiltonian_energy=energy(z),
-       )
-    return Transition(z, stat)
+    H = energy(z)
+    tstat = merge(
+        (n_steps=τ.n_steps,
+         is_accept=is_accept,
+         acceptance_rate=α,
+         log_density=z.ℓπ.value,
+         hamiltonian_energy=H,
+         hamiltonian_energy_error=H - H0),
+        stat(τ.integrator)
```

Comment:
Alignment.

Reply:
I think alignment here is correct. It's aligned with the open parenthesis of the previous item passed to `merge`.

Reply:
Then I guess the other one is mis-aligned. I just notice they are different but didn't look carefully.